src/common/tse.c

```diff
@@ -680,6 +689,10 @@ tse_task_post_process(tse_task_t *task)
 
```

Comment:
Do we need to replace "dsp" on line <678, 687> with "dsp_tmp"?

Reply:
you are right, thanks.